### DISCUSSION TITLE
Any plans for maven releases for Apple Silicon chips

### DISCUSSION THREAD

**Discussion by mwiewior** [apache/datafusion-comet#1014] [General]
Hello!
Please correct my if I'm wrong but according to my analyses maven releases contain only native code for Linux arm64/amd64? Are there any plans to include also dyn libs for Apple Silicon chips? If not  - then what setup of a project built on top of Comet would you recommend so that can be easily developed on Apple boxes, e.g. build comet locally with Apple support from a given tag and publish artifacts to local maven repository or maybe git submodule setup, anything smarter than that?

Thanks,
Marek

**Comment by andygrove** [comment]
Yes, we do plan on publishing jar files to Maven to support Apple silicon. It would make it easier for people to experiment with Comet on their laptops. I filed an issue to track this: https://github.com/apache/datafusion-comet/issues/1010
